src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,36 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => {
+  const { title, body, user, comments } = post;
+
+  return (
+
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          {user && <UserInfo user={user} />}
+        </p>
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      {comments.length ? <CommentList comments={comments} />
+        : (
```

Comment:
This syntax is easier to read in my opinion 
```suggestion
      {comments.length ? (
        <CommentList comments={comments} />
       ) : (
```